import-scripts/generate_cdd_attribute_tsv.py

```diff
@@ -153,7 +172,7 @@ def insert_value(key, value, normalized_column_header_name):
         elif key == PRIORITY_KEY:
             return "1"
         else: 
-            print >> sys.stderr, "No name is defined in normalized column header column... aborting"
+            print (sys.stderr, "No name is defined in normalized column header column... aborting")
```

Comment:
looks like we missed some of the whitespace adjustments for the print statements in this script